CHANGES.md

```diff
@@ -28,6 +28,14 @@ OpenSSL 3.3
 
 ### Changes between 3.2 and 3.3 [xx XXX xxxx]
 
+ * The activate configuration setting for providers in openssl.cnf has been
+   updated to require a value of [1|yes|true|on] (case insensitive) to
```

Comment:
'case insensitive' is not 100% accurate. How about 
```suggestion
   updated to require a value of [1|yes|true|on] (in lower or UPPER case) to
```

Reply:
fixed